src/main/java/io/phasetwo/keycloak/magic/representation/MagicLinkRequest.java

```diff
@@ -31,4 +31,7 @@ public class MagicLinkRequest {
 
   @JsonProperty("send_email")
   private boolean sendEmail = false;
+
+  @JsonProperty("scope")
+  private String scopes;
```

Comment:
I believe this needs a default in case it is not set in the request. Otherwise this will break backwards compatibility.

Reply:
The default value is null, i.e. no scopes specified. I will make it explicit.